src/Templates/OrchardCore.Templates.Cms.Web/OrchardCore.Templates.Cms.Web.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\OrchardCore.Build\Dependencies.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Why not netcoreapp2.1?

Reply:
https://github.com/OrchardCMS/OrchardCore/pull/2017